src/uct/api/uct.h

```diff
@@ -437,6 +442,23 @@ enum {
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief   @anchor UCT_FLUSH_FLAGS Flags that control completion semantic.
+ * 
+ * These flags control completion semantic for @ref uct_ep_flush and
+ * @ref uct_iface_flush functions.
+ */
+enum {
+    UCT_FLUSH_FLAG_LOCAL  = UCS_BIT(0), /**< Guarantees flush of local
+                                             outstanding operations, the buffers
```

Comment:
Then calling is local is really confusing. In addition we don't really defined in the document what is what (aka local vs remote). We can try to define the API in terms of visibility ?